src/librustc/hir/mod.rs

```diff
@@ -2189,7 +2189,7 @@ pub struct GlobalAsm {
 
 #[derive(RustcEncodable, RustcDecodable, Debug, HashStable)]
 pub struct EnumDef {
-    pub variants: HirVec<Variant>,
+    pub variants: HirVec<VariantKind>,
```

Comment:
`VariantKind` -> `Variant`.
(The `XxxKind` naming is generally used for and enum inside of a struct `Xxx`.)